### DISCUSSION TITLE
imageResizeMode on CuratorPicker

### DISCUSSION THREAD

**Discussion by MikhailFerreira** [awcodes/filament-curator#224] [Ideas]
Would it be possible to add `imageResizeMode` to the `CuratorPicker` modal and maybe default setting for `Curator` facade please?

So that there is a way to resize larger images during the upload process without having to create a separate curation later on?

I don't think `imageResizeTargetWidth` and `imageResizeTargetHeight` works without `imageResizeMode` on the Filament `FileUpload` field.

**Comment by MikhailFerreira** [comment]
I've also noticed that the thumbnails on the `CuratorPicker` are using the original size. In the case of really large photos that cannot be resized during upload, and using a repeater or a `multiple` picker, the form would take very long to load.

Also, maybe a `maxItems` method might also be great on a `multiple` picker.

Any update on this would be hightly appreciated.

**Comment by awcodes** [comment]
Added in v2.8.0

**Comment by MikhailFerreira** [comment]
Amazing! Thanks so much.
